packages/google_maps_routes_api/lib/src/types/enums.dart

```diff
@@ -0,0 +1,182 @@
+enum Maneuver {
```

Comment:
Need to add comment for each of these values. These does not match dart formatting so probably needs to be renamed correctly. Probably if you delete that analysis file then the IDE will highlight all the places what needs comments etc.

Reply:
> These does not match dart formatting so probably needs to be renamed correctly

Do you mean the enum values? Dart would like them to be in camel case, but it would be very tedious to write the logic for matching the values coming from the API if the enum values are in camel case

Reply:
Yeah, I'm working on doing these conversions... :S Might be good to ask @jokerttu if same rules apply for these packages as well as plugins.